SourceGitlab/SourceGitlab.php

```diff
@@ -205,6 +205,9 @@ public function precommit() {
 		}
 
 		$t_data = json_decode( $f_payload, true );
+		if ( is_null( $t_data ) ) {
```

Comment:
In fact this is only a copy past of a similar check just 5 lines above (by the way, most of the file is not compliant), but if you prefer I can change this.